experimental/content-type-screen-effect/src/ScreenEffect.ts

```diff
@@ -0,0 +1,60 @@
+import { ContentTypeId } from "@xmtp/xmtp-js";
+import type { ContentCodec, EncodedContent } from "@xmtp/xmtp-js";
+
+
+export enum EffectType {
+  SNOW = "SNOW",
+  RAIN = "RAIN",
+}
+
+export const ContentTypeScreenEffect = new ContentTypeId({
+  authorityId: "xmtp.chat",
```

Comment:
should this be .org or should each app that adds a content type set their app name as the authority id?

Reply:
i think this should be `xmtp.org` unless we want to use a different org for experimental content types.